packages/@aws-cdk/aws-ses/lib/configuration-set-event-destination.ts

```diff
@@ -0,0 +1,279 @@
+  /**
+   * The configuration set that contains the event destination.
+   */
+  readonly configurationSet: IConfigurationSet;
+}
+
+/**
+ * Email sending event
+ */
+export enum EmailSendingEvent {
+  /**
+   * The send request was successful and SES will attempt to deliver the message
+   * to the recipient's mail server. (If account-level or global suppression is
+   * being used, SES will still count it as a send, but delivery is suppressed.)
+   */
+  SEND = 'send',
+
+  /**
+   * SES accepted the email, but determined that it contained a virus and didn’t
+   * attempt to deliver it to the recipient’s mail server.
+   */
+  REJECT = 'reject',
+
+  /**
+   * (Hard bounce) The recipient's mail server permanently rejected the email.
+   * (Soft bounces are only included when SES fails to deliver the email after
+   * retrying for a period of time.)
+   */
+  BOUNCE = 'bounce',
+
+  /**
+   * The email was successfully delivered to the recipient’s mail server, but the
+   * recipient marked it as spam.
+   */
+  COMPLAINT = 'complaint',
+
+  /**
+   * SES successfully delivered the email to the recipient's mail server.
+   */
+  DELIVERY = 'delivery',
+
+  /**
+   * The recipient received the message and opened it in their email client.
+   */
+  OPEN = 'open',
+
+  /**
+   * The recipient clicked one or more links in the email.
+   */
+  CLICK = 'click',
+
+  /**
+   * The email wasn't sent because of a template rendering issue. This event type
+   * can occur when template data is missing, or when there is a mismatch between
+   * template parameters and data. (This event type only occurs when you send email
+   * using the `SendTemplatedEmail` or `SendBulkTemplatedEmail` API operations.)
+   */
+  RENDERING_FAILURE = 'renderingFailure',
+
+  /**
+   * The email couldn't be delivered to the recipient’s mail server because a temporary
+   * issue occurred. Delivery delays can occur, for example, when the recipient's inbox
+   * is full, or when the receiving email server experiences a transient issue.
+   */
+  DELIVERY_DELAY = 'deliveryDelay',
+
+  /**
+   * The email was successfully delivered, but the recipient updated their subscription
+   * preferences by clicking on an unsubscribe link as part of your subscription management.
+   */
+  SUBSCRIPTION = 'subscription'
+}
+
+/**
+ * A CloudWatch dimension upon which to categorize your emails
+ */
+export interface CloudWatchDimension {
+  /**
+   * The place where Amazon SES finds the value of a dimension to publish to
+   * Amazon CloudWatch.
+   */
+  readonly source: CloudWatchDimensionSource;
+
+  /**
+   * The name of an Amazon CloudWatch dimension associated with an email sending metric.
+   */
+  readonly name: string;
+
+  /**
+   * The default value of the dimension that is published to Amazon CloudWatch
+   * if you do not provide the value of the dimension when you send an email.
+   */
+  readonly default: string;
```

Comment:
I'm wondering if calling this `defaultValue` would be more clear.

Reply:
Changed to `defaultValue`.